Makefile

```diff
@@ -40,3 +40,6 @@ check-fmt:
 
 .PHONY: gen
 gen: .gen-op .gen-errdetails
+
+atlas-cli:
```

Comment:
should this be called `build-cli` or something? There's no verby thing to indicate to me that it does outside of reading the command